composer/loggers/console_logger.py

```diff
@@ -57,6 +57,7 @@ def __init__(self,
             else:
                 raise ValueError(f'stream must be one of ("stdout", "stderr", TextIO-like), got {stream}')
         self.should_log_traces = log_traces
+        self.eval_log_interval = Time.from_timestring('100ba')
```

Comment:
I'm thinking this should probably not be hardcoded?

Reply:
moved to deciles.